macro/o2sim.C

```diff
@@ -25,13 +25,27 @@
 #include <SimSetup/SimSetup.h>
 #include <Steer/O2RunSim.h>
 #include <DetectorsBase/MaterialManager.h>
+#include <CCDB/BasicCCDBManager.h>
 #include <unistd.h>
 #include <sstream>
 #endif
 
 FairRunSim* o2sim_init(bool asservice)
 {
   auto& confref = o2::conf::SimConfig::Instance();
+  // initialize CCDB service
+  auto& ccdbmgr = o2::ccdb::BasicCCDBManager::instance();
+  ccdbmgr.setURL(confref.getConfigData().mCCDBUrl);
+  ccdbmgr.setTimestamp(confref.getConfigData().mTimestamp);
```

Comment:
Note that currently GRP set run id extracted from the  ``FairRunSim* run``: https://github.com/AliceO2Group/AliceO2/blob/f65c05b0a8fa4aa781f75cc9772321933d736f10/macro/o2sim.C#L135
By default, this time stamp also generated as "now" if was not set beforehand to some value. For the consistency, I would also set the time-stamp to FairRunSim by adding before ``run->Init()``  the line
````
run->SetRunId(confref.getConfigData().mTimestamp);
````
e.g. here: https://github.com/AliceO2Group/AliceO2/blob/b54ed865fc9dd6fb7b19fb762e58d7a4d83735f8/macro/o2sim.C#L72

then it would propagate to the GRP,

Reply:
Good idea.